### PR TITLE
vendor c/common

### PR DESCRIPTION
**Pull request by vrothberg**
Update the recent events-log changes to fix the build error.

Signed-off-by: Valentin Rothberg <vrothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@nicrowe00 @Luap99 PTAL

**Review by Luap99 — COMMENTED** [review]
LGTM

**Comment by Luap99** [comment]
Looks like something with the event change is wrong.

**Comment by vrothberg** [comment]
Ah, I know what's going.

**Comment by vrothberg** [comment]
> Ah, I know what's going.

Should pass now. `os.ErrNotExist` was returned if the events file did not exist which was wrong.

File: libpod/events/logfile.go [hunk 1]
```diff
@@ -158,6 +158,10 @@ func rotateLog(logfile string, content string, limit uint64) (bool, error) {
 	}
 	file, err := os.Stat(logfile)
 	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			// The logfile does not exist yet.
+			return false, nil
+		}
```
Comment (Luap99) [inline libpod/events/logfile.go:164]: I think this hides another problem. We didn't need this before and now it fails. That means something changed in c/common. I assume we do not get the default value want. I assume it is 0 for some reason.

Comment (vrothberg) [reply libpod/events/logfile.go:164]: I don't think it's hiding a problem.  The default has changed from 0 to 1mb, so we did not hit the ErrNotExist case before.

Comment (Luap99) [reply libpod/events/logfile.go:164]: Why would it try to rotate a log which doesn't exists yet? This sounds very dangerous.

Comment (vrothberg) [reply libpod/events/logfile.go:164]: I agree and now we don't rotate it anymore.

Comment (Luap99) [reply libpod/events/logfile.go:164]: Ok I understand it now, we always call rotateLog() before we even write an event. So of course the file cannot exists yet and we need to ignore this.

**Review by Luap99 — COMMENTED** [review]
LGTM

**Review by flouthoc — APPROVED** [review]
LGTM
/lgtm
/hold

**Pull request merged**